content/id/examples/pods/pod-single-configmap-env-variable.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dapi-test-pod
+spec:
+  containers:
+    - name: test-container
+      image: k8s.gcr.io/busybox
+      command: [ "/bin/sh", "-c", "env" ]
+      env:
+        # Define the environment variable
+        - name: SPECIAL_LEVEL_KEY
+          valueFrom:
+            configMapKeyRef:
+              # The ConfigMap containing the value you want to assign to SPECIAL_LEVEL_KEY
+              name: special-config
+              # Specify the key associated with the value
+              key: special.how
```

Comment:
If you want, you can localize these comments

Reply:
Be sure to update the URLs of the examples to match.

Reply:
Oh, yeah, I was wondering if I should translate the comments on the sample files, so I think will do that for now. Thank you @sftim